pages/home_page.py

```diff
@@ -24,6 +26,13 @@ def __init__(self, testsetup, product=None):
 
         if product is None:
             self.selenium.get(self.base_url)
+        self.wait_for_page_to_load()
+
+    def wait_for_page_to_load(self):
+        Wait(self.selenium, self.timeout).until(
+            lambda s: 'loading' not in s.find_element(
+                *self._graph_locator).get_attribute('class'))
+        return self
```

Comment:
I like this approach. One additional question, have we standardized on moving the `wait_for_page_to_load()` out of the constructor?